### PR TITLE
[+] Authentication - Users can connect to their project using Google Single Sign-On

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by ArnaudValensi** [comment]
[(21) SSO / OAuth support](https://trello.com/c/fuuv6C98/2046-sso-oauth-support)

**Comment by ArnaudValensi** [comment]
I've added Babel for the tests. The next step to have a clean code is to build the source with babel to a build folder on deploy.

File: package.json [hunk 1]
```diff
@@ -40,16 +40,26 @@
     "winston": "2.2.0"
   },
   "devDependencies": {
+    "babel-core": "6.26.0",
```
[the remark this answers is not in the document]
Comment (ArnaudValensi) [reply package.json]: The PR of the next TGIFA will add es6 to all the source and add a proper linting check with eslint.

File: test/routes/sessions-google.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            last_name: 'LastName',
+            teams: [],
+          });
+        };
+      },
+
+      ApimapSender: function () {
+        this.perform = function () {};
+      }
+    };
```
[the remark this answers is not in the document]
Comment (ArnaudValensi) [reply test/routes/sessions-google.js]: I tryed again with no success :/

File: services/google-authorization-finder.js [hunk 3]
```diff
@@ -5,15 +5,15 @@ var logger = require('./logger');
...
+function GoogleAuthorizationFinder(renderingId, forestJwt, envSecret) {
   this.perform = function () {
     return new P(function (resolve, reject) {
       var forestUrl = new ServiceUrlGetter().perform();
 
       request
         .get(forestUrl + '/forest/renderings/' + renderingId + '/google-authorization')
         .set('forest-secret-key', envSecret)
-        .set('google-access-token', accessToken)
+        .set('forest-jwt', forestJwt)
```
Comment (arnaudbesnier) [inline services/google-authorization-finder.js]: `forest-token` is a better name. Whatever the technology we use, this is a token.

**Pull request merged**